plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2036,63 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class RelaxationRates:
```

Comment:
Personally I would change this to 'CollisionFrequencies': I hear that term more than relaxation rates.

Reply:
Is calling them "relaxation parameters" in the changelog still acceptable in addition to referencing the function?

Reply:
I'd call them collision frequencies there too